guides/source/form_helpers.md

```diff
@@ -505,93 +507,77 @@ The first parameter specifies which value should be selected and can either be a
 
 will produce the same output and the value chosen by the user can be retrieved by `params[:date][:year]`.
 
-Creating Checkboxes for Relations
-------------------------------------------
+Choices from a Collection of Arbitrary Objects
+----------------------------------------------
 
-Generating checkboxes options that act appropriately with Active Record models
-and strong parameters can be a confusing undertaking. Luckily there is a helper
-function to significantly reduce the work required.
+Often, we want to generate a set of choices in a form from a collection of objects. For example, when we want the user to choose from cities in our database, and we have a `City` model like this:
```

Comment:
I am trying to understand the reasoning here.  Since this is a completely new paragraph (from scratch), it cannot really be compared to the old paragraph.  In other words, breaking it up into multiple lines to mimic the old paragraph will not improve the diff.

Reply:
I think it comes from this:

![image](https://user-images.githubusercontent.com/567626/83923716-73586400-a7a0-11ea-9d71-02edad84ebfc.png)

Most text editors don't wrap on lines, so reading longer sentences and editing can be harder.  Given that this file already uses long line, can't really say its the default, just "good to have"